docker/Dockerfile

```diff
@@ -1,11 +1,23 @@
 FROM ubuntu:24.04
 
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update && apt-get install --yes git build-essential pkg-config libcjson-dev openjdk-17-jdk gradle libbcprov-java libgoogle-gson-java libssl-dev gcc g++ libbotan-2-dev nlohmann-json3-dev python3 python3-pip rustc cargo iputils-ping telnet wget && rm -fr /var/cache/apt/* /var/lib/apt/lists/*
+RUN apt-get update && apt-get install --yes curl wget zip git build-essential pkg-config libcjson-dev openjdk-17-jdk libbcprov-java libgoogle-gson-java libguava-java libssl-dev gcc g++ libbotan-2-dev nlohmann-json3-dev python3 python3-pip rustc cargo iputils-ping telnet wget && rm -fr /var/cache/apt/* /var/lib/apt/lists/*
 RUN pip3 install --break-system-packages cryptography requests
 
 RUN GO_VERSION=$(wget -qO- https://go.dev/VERSION?m=text | head -n 1) && wget https://go.dev/dl/${GO_VERSION}.linux-amd64.tar.gz && rm -rf /usr/local/go && tar -C /usr/local -xzf ${GO_VERSION}.linux-amd64.tar.gz && rm -rf ${GO_VERSION}.linux-amd64.tar.gz
 ENV PATH $PATH:/usr/local/go/bin
 
 COPY rust /rust
 RUN cd /rust && cargo vendor
+
+# Install SDKMAN
+RUN curl -s "https://get.sdkman.io" | bash
+# Install Gradle
+RUN bash -c "source $HOME/.sdkman/bin/sdkman-init.sh && sdk install gradle"
+
+# Installing gradle plugin dependecy...
+COPY gradle /gradle
```

Comment:
Gradle ins root-Verzeichnis? Nein.

Reply:
Ich habe nur das was auch bei Rust gemacht wird nachgeahmt, der Ordner der für diesen Part hier verwendet wird ist aber irrelevant und kann nach dem ausführen meines Workarounds auch ebenfalls gelöscht werden

Reply:
Ja genau, das ist für die temporären Dateien. Gradle selbst landet in `$HOME/.sdkman/candidates/gradle/current/bin/gradle`, wenn man es per SDKMAN installiert

Man kann Gradle auch direkt installieren (Binary herunterladen), dann kann man natürlich /usr/local/bin verwenden. Ich hatte SDKMAN gelassen, weil ich zuvor einige Stunden am verzweifeln war, das Environment zum laufen zu bringen...